docs/quickstart.md

```diff
@@ -2,7 +2,9 @@
 
 The following example sets up the Ark server and client, then backs up and restores a sample application.
 
-For simplicity, the example uses Minio, an S3-compatible storage service that runs locally on your cluster. See [Set up Ark with your cloud provider][3] for how to run on a cloud provider. 
+For simplicity, the example uses Minio, an S3-compatible storage service that runs locally on your cluster. Backups and restores for an on-premises cluster work the same way as the example.
```

Comment:
I think it would better call out attention to what we are trying to say if: `s/work the same way as the example/work the same way as with cloud providers`

Reply:
I kept the sentence about cloud providers but moved it to a new paragraph. The idea was to address a customer issue (#332) where there was confusion about how to run Ark on-prem. Do you think it might work better to switch two around (revert this paragraph to its original and add a one-sentence paragraph about on-prem)?

Reply:
Hm. How about this: We could make the very first line: `The example below for backups and restores works the same for on-premises and cloud provider clusters.`

Then revert to the original. Wdyt?